apps/vaporgui/TFWidget.cpp

```diff
@@ -268,6 +288,23 @@ void TFWidget::getVariableRange(
 	values[1] = tf->getMaxMapValue();
 }
 
+void TFWidget::calculateStride( string varName ) {
+    std::vector<size_t> dimsAtLevel;
+	int ref = _rParams->GetRefinementLevel();
+    int rc = _dataMgr->GetDimLensAtLevel(varName, ref, dimsAtLevel);
+    assert(rc>=0);
+
+    int size = 1;
```

Comment:
An `int` can represent the biggest value around 1000x1000x2000 . What will this code do when the grid is larger than that?